convert_case.py

```diff
@@ -1,13 +1,14 @@
-message = input("Please write the string you want to case-convert: ")
-option = input("Please choose from the following options: \n \
+message = str(input("Please write the string you want to case-convert: "))
+option = str(input("Please choose from the following options: \n \
```

Comment:
```suggestion
option = input("Please choose from the following options: \n \
```

@Pianomen Same here. No need for string conversion, imo. 🙂

Reply:
Hmm, agree , bit overthinking the moment, you right :)